spec/classes/init_spec.rb

```diff
@@ -672,4 +672,41 @@
       end
     end
   end
+
+  describe 'with disable_monitor set' do
+    let(:facts) { { :osfamily => 'RedHat' } }
+
+    [true,'true',false,'false'].each do |value|
+
+      if !!value == value
```

Comment:
This is clever, though I'm not sure how it actually works. Could we use `value.class` here?